TodoList/src/constants/routes.js

```diff
@@ -1,5 +1,4 @@
 export default {
-  Books: 'BOOKS',
```

Comment:
Revert this as it was before. Does the navigator work without the `Books` route?